Marlin/src/pins/lpc1768/pins_BTT_SKR.h

```diff
@@ -63,15 +66,15 @@
 #endif
 #if HOTENDS == 1
   #ifndef FAN1_PIN
-    #define FAN1_PIN       P2_04
+    #define FAN1_PIN       P2_03 // Swapped due to different plugs
   #endif
 #else
   #ifndef HEATER_1_PIN
     #define HEATER_1_PIN   P2_04
   #endif
 #endif
 #ifndef FAN_PIN
-  #define FAN_PIN          P2_03
+  #define FAN_PIN          P2_04 // Swapped due to different plugs
```

Comment:
Should this patch be applied now for all users of this board, or does this only apply when `TEMP_SENSOR_PROBE` is in use?

Reply:
It doesn’t make sense to swap these pins in this PR/seems like a user preference.

Reply:
sry, the swap is just for my personal setup of wiring. I missed to revert that before commiting.